app/src/main/res/values/strings.xml

```diff
@@ -93,4 +93,14 @@
     <string name="firstrun_description">Automatically block online trackers while you browse. Then tap to erase visited pages, searches, cookies and passwords from your device.</string>
     <string name="firstrun_button">Ok, got it!</string>
     <string name="firstrun_footer">To be clear, sites and service you use can still know where you\'ve been.</string>
+
+    <!-- This string is shown when the user has clicked a link that needs to be opened in a different app.
+         Argument 2 is the external app name, argument 1 is our name (i.e. Firefox Focus, or Firefox Klar). -->
+    <string name="external_app_prompt">This link will open in %2$s. Are you sure you want to exit %1$s?</string>
+    <string name="external_app_prompt_no_app_title">No app to handle link</string>
+    <string name="external_app_prompt_no_app">None of the apps on your device are able to open this link. Do you want to exit %1$s and open %2$s to find a suitable app?</string>
+    <string name="external_app_prompt_ok">Ok</string>
+    <string name="external_app_prompt_cancel">Cancel</string>
```

Comment:
I also added a 'Cancel' string for a dialog in one of my patches that I'm currently landing. We should make sure to unify them and just have one string before exporting them again.